src/Core/View.js

```diff
@@ -319,9 +319,14 @@ View.prototype.addLayer = function addLayer(layer, parentLayer) {
         layer.update = layer.update || updateLayeredMaterialNodeElevation;
     }
     return new Promise((resolve, reject) => {
+        if (!layer) {
```

Comment:
Oh we missed that: this is dead code, if layer is undefined, it'd throw on line 316 before reaching this.